docs/DeveloperGuide.adoc

```diff
@@ -767,6 +817,48 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Create a contact
+
+*MSS*
+
+1. User inputs the command with the correct keywords
+2. AddressBook confirms the addition by showing the confirmation message
++
+Use case ends.
+
+*EXTENSIONS*
+
+* 2a. The given input is invalid (Wrong format, etc)
++
+** 2a1. AddressBook shows an error message
++
+Use case ends.
+
+[discrete]
+=== Use case: Edit/Update a specific contact
+
+*MSS*
+
+1. User requests to edit a contact and inputs the command with correct keywords
```

Comment:
Since MSS is the main success scenario, there is no need to specify `correct keywords`.